distribution/tools/plugin-cli/src/main/java/org/elasticsearch/plugins/cli/ListPluginsCommand.java

```diff
@@ -61,14 +61,14 @@ private void printPlugin(Environment env, Terminal terminal, Path plugin, String
         terminal.println(Terminal.Verbosity.SILENT, prefix + plugin.getFileName().toString());
         PluginDescriptor info = PluginDescriptor.readFromProperties(env.pluginsFile().resolve(plugin));
         terminal.println(Terminal.Verbosity.VERBOSE, info.toString(prefix));
-        if (info.getElasticsearchVersion().equals(Version.CURRENT) == false) {
+        if (info.getElasticsearchVersion().toString().equals(Build.current().version()) == false) {
             terminal.errorPrintln(
                 "WARNING: plugin ["
                     + info.getName()
                     + "] was built for Elasticsearch version "
                     + info.getElasticsearchVersion()
                     + " but version "
-                    + Version.CURRENT
+                    + Build.current().unqualifiedVersion()
```

Comment:
I would be consistent here: either use `Build.current().unqualifiedVersion()` in the comparison or output `Build.current().version()` (otherwise we may get a very confusing message like "was build for 8.11 but version 8.11 is required". I would stick to `Build.current().version()` 

Even better: here and in the other cases, until we get to PluginDescriptor, it might be worth having a `getElasticsearchVersion()` helper that adds the -SNAPSHOT suffix to the Version. We will probably remove it when we move away from Version in PluginDescriptor later.